hardhat.config.ts

```diff
@@ -45,6 +45,15 @@ BigInt.prototype.toSharesDown = function (totalAssets: BigNumberish, totalShares
   return toSharesDown(this as bigint, totalAssets, totalShares);
 };
 
+const commonCompilerSettings : any = {
+  optimizer: { 
+    enabled: true,
+    runs: 200,
```

Comment:
For Morpho Blue, we use 999999 optimizer runs

Reply:
We don't care because we trust the compiler so we can assume it's functionally the same, but hardhat runs gas tests so I think we care here

Reply:
Fine!

Reply:
@adhusson this discussion got resolved but no changes were pushed. I think @QGarchery suggested to not use the same settings between 0.8.21 and 0.8.19

Reply:
I misunderstood my bad, thanks for spotting this. Pushed 880db7f7b074c5f7a88c6583180c01bcc3d43096.